evalml/pipelines/utils.py

```diff
@@ -213,6 +214,10 @@ def make_pipeline(
     Raises:
         ValueError: If estimator is not valid for the given problem type, or sampling is not supported for the given problem type.
     """
+    problem_type = handle_problem_types(problem_type)
+    if is_clustering(problem_type):
+        raise ValueError("Pipelines for clustering problems are not supported yet")
+
     X = infer_feature_types(X)
     y = infer_feature_types(y)
 
```

Comment:
You should be able to remove the call to `problem_type = handle_problem_types(problem_type)` below in the `if estimator` call since you now define it up top.